server/app.py

```diff
@@ -32,7 +33,7 @@ def create_app():
 
     # Database variables
     user = 'postgres'
-    pw = 'P@ssw0rd'
+    pw = '123456'
```

Comment:
Haha, we keep changing this, we definitely want to tackle that PR to put this in environment variables